dcc6502.c

```diff
@@ -613,6 +614,7 @@ static void usage(void) {
     fprintf(stderr, "  -?/-h        : Show this help message\n");
     fprintf(stderr, "  -o ORIGIN    : Set the origin (base address of disassembly) [default: 0x8000]\n");
     fprintf(stderr, "  -m NUM_BYTES : Only disassemble the first NUM_BYTES bytes\n");
+    fprintf(stderr, "  -s NUM_BYTES : Disassemble starting from NUM_BYTES input file offset\n");
```

Comment:
A more general description could be: 
`"  -s NUM_BYTES : Disassemble after skipping NUM_BYTES from start of input file\n"`

It does not use the technical "offset" term.